docs-preview/src/xref/xref.ts

```diff
@@ -74,7 +74,9 @@ async function updateXrefContent(md: any, src: string) {
   }
 
   while ((captureGroup = XREF_MD_LINK_RE.exec(src))) {
-    const uid = captureGroup[1].trim();
+    const uidWithParams = captureGroup[1].trim();
+    const uid = uidWithParams.split("?")[0];
+    console.log(captureGroup);
```

Comment:
@bharney0 , looks good.  We can probably remove the console.log but the functionality is great.

Reply:
@jamarw Removed the console.log. Thanks for letting me know.